driver-service/src/main/java/com/vlad/kuzhyr/driverservice/utility/broker/DriverEventListener.java

```diff
@@ -30,4 +33,19 @@ public void consumeDriverBusyEvent(
 
         driverProcessor.updateDriverByIdAndIsBusy(driverId, isBusy);
     }
+
+    @KafkaListener(
+        topics = "${spring.kafka.topic.driver-create-topic}",
+        groupId = "driver-auth-group"
+    )
+    public void consumeDriverCreateTopic(
+        String message
+    ) {
```

Comment:
Нет смысла разбивать так строку, поскольку она не превышает 120 символов.
```suggestion
    public void consumeDriverCreateTopic(String message) {
```